test/spec/file-handler.spec.js

```diff
@@ -6,10 +6,16 @@ import sinon from "sinon";
 
 
 describe("fileHandler", () => {
+  const sandbox = sinon.sandbox.create();
```

Comment:
Never "do" anything outside of `beforeEach|afterEach|it`, just set scope. Should change to:

```js
let sandbox;

beforeEach(() => {
  sandbox = sinon.sandbox.create();
});
```